deploy/docs/v2_migration_doc.md

```diff
@@ -166,3 +219,11 @@ to convert their existing `values.yaml` file into one that is compatible with th
   ```bash
   helm upgrade collection sumologic/sumologic --version=2.0.0 -f new_values.yaml
   ```
+
+### 5. Remove temporary instance of fluent-bit
+
+Remove the temporary copy of fluent-bit DaemonSet to avoid of data duplication in longer period:
+
+```bash
+kubectl delete daemonset -l "app=fluent-bit,heritage=tmp,release=<RELEASE-NAME>"
+```
```

Comment:
I would move it as  `#### 3.1` or sth like that. IMHO its more intuitive to keep it in the fluent-bit upgrade section